SDK/AppCenterDistribute/Microsoft.AppCenter.Distribute.Android/Distribute.cs

```diff
@@ -51,6 +51,30 @@ static void HandleUpdateAction(UpdateAction updateAction)
             }
         }
 
+        static void SetUpdateTrack(UpdateTrack updateTrack)
```

Comment:
This doesn't handle the invalid track properly. If the caller passes an invalid track, the switch statement won't do anything because there is no matching case or default case for the switch. But we want to log an error on invalid value.

The android sdk handles invalid value and does logging properly. Either pass through the invalid value to native sdk, OR add a default case to the switch statement to handle invalid value.

Reply:
Just pass the int directly honestly, a c# enum can be cast to int